zebra/zebra_ptm.c

```diff
@@ -589,6 +589,8 @@ static int zebra_ptm_handle_msg_cb(void *arg, void *in_ctxt)
 	char port_str[128];
 	char cbl_str[32];
 	char cmd_status_str[32];
+	char vrf_str[64];
```

Comment:
How have you tested this particular patch?  Can you provide me the details?

Reply:
actually, I did not test it, as it requires to use external ptm daemon.
I did not make some patch with PTM, but I think some changes should be needed:
```
src/ptm_event.h:#define MAXNAMELEN 255
src/ptm_bfd.h:  char vrf_name[MAXNAMELEN+1]; // from struct ptm_bfd_mhop_key
src/ptm_bfd.h:    char name[MAXNAMELEN+1]; // from bfd_vrf
```
what I can see is that even before, the values were not good, as there was a risk of buffer overflow in frr. here, the problem is still present with ptm, but at least, bfd daemon from frr is aligned with the frr internals.